.buildkite/pipeline_cpu_template.py

```diff
@@ -30,6 +30,8 @@ class BkStep(str, Enum):
             "c5n.metal",
             "m5n.metal",
             "m6i.metal",
+            "m7i.metal-24xl",
```

Comment:
is there value to doing both? do the CPUs differ?

Reply:
we keep both until performance evaluation has been done :)